pull_request_template.md

```diff
@@ -1,65 +1,29 @@
-> include any environmental setup steps that aren't in the normal README steps
-> and/or any time-based elements that this requires.
-
----
-
-> Étapes consécutives (1., 2., 3., …) qui décrivent la façon de tester la
-> modification. Elles aideront les développeurs à faire des tests sans avoir à
-> jouer au détective. Veuillez aussi inclure toutes les étapes de configuration
-> de l’environnement qui ne font pas partie des étapes normales dans le fichier
-> README et tout élément temporel requis.
+_TODO: Fill in test instructions for the reviewer._
 
 # Release Instructions | Instructions pour le déploiement
 
 None.
 
-> Necessary steps to perform before and after the deployment of these changes.
-> For example, emptying the cache on a feature that changes the cache data
-> structure in Redis could be mentioned.
-
----
-
-> Étapes nécessaires à exécuter avant et après le déploiement des changements
-> introduits par cette proposition. Par exemple, vider la cache suite à des
-> changements modifiant une structure de données de la cache pourrait être
-> mentionné.
-
 # Reviewer checklist | Liste de vérification du réviseur
 
 This is a suggested checklist of questions reviewers might ask during their
 review | Voici une suggestion de liste de vérification comprenant des questions
 que les réviseurs pourraient poser pendant leur examen :
 
-
-- [ ] Is the code maintainable? | Est-ce que le code peut être maintenu?
-- [ ] Have you tested it? | L’avez-vous testé?
-- [ ] Are there automated tests? | Y a-t-il des tests automatisés?
-- [ ] Does this cause automated test coverage to drop? | Est-ce que ça entraîne
-      une baisse de la quantité de code couvert par les tests automatisés?
-- [ ] Does this break existing functionality? | Est-ce que ça brise une
-      fonctionnalité existante?
-- [ ] Does this change the privacy policy? | Est-ce que ça entraîne une
-      modification de la politique de confidentialité?
-- [ ] Does this introduce any security concerns? | Est-ce que ça introduit des
-      préoccupations liées à la sécurité?
-- [ ] Does this significantly alter performance? | Est-ce que ça modifie de
-      façon importante la performance?
-- [ ] What is the risk level of using added dependencies? | Quel est le degré de
-      risque d’utiliser des dépendances ajoutées?
-- [ ] Should any documentation be updated as a result of this? (i.e. README
-      setup, etc.) | Faudra-t-il mettre à jour la documentation à la suite de ce
-      changement (fichier README, etc.)?
+- [ ] Check that the current PR does not break existing functionality.
+- [ ] Check that this does not violate GCNotify's privacy policies.
+- [ ] Check that this does not raise any security concern.
+- [ ] Check that this does not significantly alter performance.
+- [ ] Check if additional documentation needs to be updated as a result of these changes such
+      as the README, setup instructions, a related ADR or the technical documentation.

```

Comment:
Small difference, but for these I was thinking of using the form:
- [ ] This PR does not break existing functionality
- [ ] This PR does not violate GCNotify's privacy policies
etc.

Then they are geared more torwards action/sign-off. Thoughts?

Reply:
Yeah I would like it to be geared toward action and sign off items as well. The original one for the first box was `Does this break existing functionality?` which was more of a yes/no question initially rather than an action item. The current submitted form is tedious whereas yours is more to the point. Let me change these to be aligned with your suggestions.